plugins/Denizen/scripts/fun/forbidden_words.dsc

```diff
@@ -0,0 +1,72 @@
+get_forbidden_word:
+    type: procedure
+    debug: false
+    definitions: item|should_exist
+    script:
+    - if <[should_exist]> and not <server.has_flag[forbidden_words]>:
+        - determine null
+    - if <[item].material.name> != diamond or !<[item].has_display>:
+        - determine null
+    - define word <[item].display.strip_color.to_lowercase.before[ ]>
```

Comment:
to_lowercase isn't needed. This output doesn't display anywhere, and data values are case-insensitive